var/spack/repos/builtin/packages/swig/package.py

```diff
@@ -31,12 +31,29 @@ class Swig(AutotoolsPackage):
     version('2.0.12', 'c3fb0b2d710cc82ed0154b91e43085a4')
     version('2.0.2',  'eaf619a4169886923e5f828349504a29')
     version('1.3.40', '2df766c9e03e02811b1ab4bba1c7b9cc')
+    version('master',
```

Comment:
Can you move `master` above all of the stable releases? We like to sort from newest to oldest.

Reply:
Sure thing, but I thought that would make it the 'preferred' version, whereas I would think the latest officially released version should be preferred.

Should the fortran fork also be above the 'released' versions?

Reply:
> but I thought that would make it the 'preferred' version

The preferred version is always the newest numerical release, unless `preferred=True` is added. The order of versions in the package does not affect concretization.

> Should the fortran fork also be above the 'released' versions?

I would probably leave that at the bottom. Spack sorts all non-develop/master versions as older than every other version.

Reply:
Got it. Thanks for the clarification!